inputFiles/phaseField/PhaseFieldFracture_SpectralSplit.xml

```diff
@@ -48,21 +48,33 @@
   </Solvers>
 
   <Mesh>
-    <PAMELAMesh
+    <VTKMesh
       name="Mode2Notched"
-      file="Mode2CrackMesh/geo.msh"/>
-
-    <!-- <InternalMesh name="mesh1" -->
-    <!--               elementTypes="{C3D8}" -->
-    <!--               xCoords="{0, 1}" -->
-    <!--               yCoords="{0, 1}" -->
-    <!--               zCoords="{0, 0.1}" -->
-    <!--               nx="{5}" -->
-    <!--               ny="{5}" -->
-    <!--               nz="{1}" -->
-    <!--               cellBlockNames="{all_HEX}"/> -->
+      file="notchedMesh.vtk"/>
   </Mesh>
 
+  <Geometry>
+    <Box
+      name="top"
+      xMin="{ -0.01,  0.99, -0.01 }"
+      xMax="{  1.01,  1.01,  0.11 }"/>
+
+    <Box
+      name="bottom"
+      xMin="{ -0.01, -0.01, -0.01 }"
+      xMax="{  1.01,  0.01,  0.11 }"/>
+
+    <Box
+      name="left"
+      xMin="{ -0.01, -0.01, -0.01 }"
+      xMax="{  0.01,  1.01,  0.11 }"/>
+
+    <Box
+      name="right"
+      xMin="{  0.99, -0.01, -0.01 }"
+      xMax="{  1.01, 1.01,  0.11 }"/>
+  </Geometry>
```

Comment:
Why do you have to add this?

Reply:
`top`, `bottom`, `left` and `right` were attributes associated to faces in the original `msh` file. In the `vtk ` mesh I got rid of cell attributes and simply kept three-dimensional cells with the above boxes introduced to identify boundary nodes.